README.md

```diff
@@ -189,19 +195,19 @@ An easy example would be to subscribe to changes of the master store, since that
 class Connection extends Component {
```

Comment:
I don't think this should be in the readme since it presents internals that may confuse the user about the usable API